llm_handler.py

```
@@ -141,6 +141,13 @@ async def call_llm_for_summary(messages, channel_name, date, hours=24):
         # Join the messages with newlines
         messages_text = "\n".join(formatted_messages_text)
 
+        # Truncate input if it's too long to avoid token limits
+        # Rough estimate: 1 token ≈ 4 characters, leaving room for prompt and response
+        max_input_length = 60000  # ~15k tokens for input, allowing room for system prompt and output
+        if len(messages_text) > max_input_length:
+            messages_text = messages_text[:max_input_length] + "\n\n[Messages truncated due to length...]"
+            logger.info(f"Truncated conversation input from {len('\n'.join(formatted_messages_text))} to {len(messages_text)} characters")
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string expression part cannot include a backslash (llm_handler, line 149)'

(E0001)